mqtt/mqtt-bridge/src/client.rs

```diff
@@ -1,23 +1,31 @@
 #![allow(dead_code)] // TODO remove when ready
-use std::{collections::HashSet, fmt::Display, time::Duration};
+use std::{
+    collections::HashSet, fmt::Display, io::Error, io::ErrorKind, pin::Pin, str, time::Duration,
+};
 
 use async_trait::async_trait;
 use chrono::Utc;
-use futures_util::future::BoxFuture;
-use tokio::{net::TcpStream, stream::StreamExt};
-use tracing::{debug, error};
+use futures_util::future::{self, try_join3, BoxFuture};
+use openssl::{ssl::SslConnector, ssl::SslMethod, x509::X509};
+use tokio::{io::AsyncRead, io::AsyncWrite, net::TcpStream, stream::StreamExt};
+use tokio_openssl::connect;
```

Comment:
we usually don't import functions from modules, and use with module name instead. it helps to avoid ambighuity: methods is in the current module or from somewhere else.

Reply:
same for try_join_3